WinUIGallery/ControlPages/EasingFunctionPage.xaml

```diff
@@ -144,8 +144,8 @@
                 <StackPanel>
                     <ComboBox x:Name="EasingComboBox" AutomationProperties.Name="Easing type" ItemsSource="{x:Bind EasingFunctions}" 
                               DisplayMemberPath="Name" SelectedValuePath="EasingFunctionBase" Loaded="EasingComboBox_Loaded"/>
-                    <RadioButtons>
-                        <RadioButton x:Name="easeOutRB" Content="EaseOut" IsChecked="True" />
+                    <RadioButtons x:Name="EasingModeRadioButtons" SelectedIndex="0">
```

Comment:
Is this related to a XAMl substition or just general improvement?

Reply:
> Is this related to a XAMl substition or just general improvement?

If you simply set `IsChecked` on a `RadioButton` instead of setting `SelectedIndex` on its parent `RadioButtons`, the substitution does not work.